### PR TITLE
add layout manager

### PR DESCRIPTION
**Pull request by kkrisstoff**
NOT FOR MERGE!!!!

Route logic was putted in separate class (LayoutManager)

### CODE REVIEW

File: modules/UI/Rotes.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* global APP */
+
+export default {
```
Comment (yanas) [inline modules/UI/Rotes.js]: The name of the js is meant to be Routes.js I guess.

Comment (kkrisstoff) [inline modules/UI/Rotes.js]: Yep

File: modules/UI/UI.js [hunk 2]
```diff
@@ -438,18 +441,31 @@ UI.getSharedVideoManager = function () {
...
     document.title = interfaceConfig.APP_NAME;
+
+    return UI.layoutManager.route(window.location.pathname);
+};
+
+
+UI.goToHome = function() {
+    console.log("GOTO: home");
     var setupWelcomePage = null;
-    if(config.enableWelcomePage && window.location.pathname == "/" &&
```
Comment (kkrisstoff) [inline modules/UI/UI.js]: It was an my idea. To move "window.location.pathname" in router. And create separate functions for a different layouts rendering.
Now I just divide UI.start function to showWelcomePage and showVideoPage by window.location.pathname

pathname == '/'        --> goToHomePage
pathname == '/[a-Z0-9]'         --> goToVideoPage
pathname == '/test-test or another incorrect path'     --> goToErrorPage

**Comment by kkrisstoff** [comment]
You can test it. But for testing we should change nginx routing. As I wrote here https://github.com/jitsi/jitsi-meet/pull/980

**Pull request closed without merging**